python/pyspark/sql/dataframe.py

```diff
@@ -1731,7 +1731,7 @@ def toDF(self, *cols):
         return DataFrame(jdf, self.sql_ctx)
 
     @since(1.3)
-    def toPandas(self):
+    def toPandas(self, strict=True):
```

Comment:
BTW, this change looks uesless when the optimization is enabled if I understood correctly. I won't add an option and make it complicated.